src/vcpkg.cpp

```diff
@@ -336,16 +320,23 @@ int main(const int argc, const char* const* const argv)
     LockGuardPtr<Metrics>(g_metrics)->track_property("error", exc_msg);
 
     fflush(stdout);
-    msg::println(msgVcpkgHasCrashed, msg::email = Commands::Contact::email());
+    msg::println(LocalizedString::from_raw("vcpkg.exe has crashed."));
```

Comment:
This PR preserves the previous behavior, but I think this should be localized?

Reply:
Side note, everyone I know from Mexico Spanglishes "has crashed" as "crasheo".

Reply:
It looks to me like this was actually localized before. 

Just this section was locked previously.
```
    msg::println(LocalizedString::from_raw("Version=").append_raw(Commands::Version::version));
    msg::println(LocalizedString::from_raw("EXCEPTION=").append_raw(exc_msg));
    msg::println(LocalizedString::from_raw("CMD="));
```